core/src/main/java/io/neow3j/transaction/Witness.java

```diff
@@ -23,7 +24,13 @@
     private VerificationScript verificationScript;
     private ScriptHash scriptHash;
 
+    /**
+     * Constructs an empty witness with a zero-valued script hash.
+     */
     public Witness() {
+        invocationScript = new InvocationScript();
+        verificationScript = new VerificationScript();
+        scriptHash = ScriptHash.ZERO;
     }
```

Comment:
Making sure that an empty witness is instantiated with empty byte arrays and a zero-valued script hash

Reply:
Good!